src/containers/Bookings.js

```diff
@@ -1,13 +1,36 @@
 import React, { Component } from "react";
 import Search from "../components/Search.js";
 import Results from "../components/Results.js";
-import bookings from "../data/fakeBookings.json";
+// import bookings from "../data/fakeBookings.json";
+let bookings;
 export default class Bookings extends Component {
   state = {
     filteredRsults: undefined,
     error: undefined,
-    numberOfBookingsFound: undefined
+    numberOfBookingsFound: undefined,
+    isLoading: true
   };
+  componentDidMount() {
+    const localUrl = `http://localhost:8080/api/hw/bookings`
+    const delayedUrl = `https://mire-hub.glitch.me/`
+    fetch(delayedUrl)
+      .then(res => {
+        if (res.status >= 200 && res.status < 300) {
+          return res;
+        } else {
+          throw new Error("HTTP error");
+        }
+      })
+      .then(res => res.json())
+      .then(({ results }) => {
+        console.log("kehtnn", results);
+        bookings = results;
+        this.setState({ isLoading: false, filteredRsults: results });
```

Comment:
One quick tip - it's probably a good idea to clear out any errors you have in state when you receive a successful response. That way you don't accidentally show an error message even if there is a successful response